werkit/aws_lambda/test_integration.py

```diff
@@ -0,0 +1,156 @@
+import os
+import base64
+import boto3
+import uuid
+import zipfile
+import tempfile
+import json
+from pprint import pprint
+
+role = "arn:aws:iam::139234625917:role/werkit-test-integration"
```

Comment:
It seems like this rols probably should be pulled from an env var. For development we could use a `.env` file along with [python-dotenv](https://pypi.org/project/python-dotenv/).